docs/tutorials/widgets/data-from-external-source.md

```diff
@@ -107,24 +107,25 @@ Finally, you need to load the `ExternalDataWidget` plugin in your `main.js` file
 import { ClassicEditor, Bold, Italic, Essentials, Heading, List, Paragraph } from 'ckeditor5';
 import ExternalDataWidget from './external-data-widget/externaldatawidget';
 import CKEditorInspector from '@ckeditor/ckeditor5-inspector';
+
 import 'ckeditor5/ckeditor5.css';
 
 ClassicEditor
-    .create( document.querySelector( '#editor' ), {
-        plugins: [ Essentials, Paragraph, Heading, List, Bold, Italic, ExternalDataWidget ],
-        toolbar: [ 'heading', 'bold', 'italic', 'numberedList', 'bulletedList', '|', 'undo', 'redo' ]
-    } )
-    .then( editor => {
-        console.log( 'Editor was initialized', editor );
-
-        CKEditorInspector.attach( { editor: 'editor' } );
-
-        // Expose for playing in the console.
-        window.editor = editor;
-    } )
-    .catch( error => {
-        console.error( error.stack );
-    } );
+	.create( document.querySelector( '#editor' ), {
+		plugins: [ Essentials, Paragraph, Heading, List, Bold, Italic, ExternalDataWidget ],
+		toolbar: [ 'heading', 'bold', 'italic', 'numberedList', 'bulletedList', '|', 'undo', 'redo' ]
+	} )
+	.then( editor => {
+		console.log( 'Editor was initialized', editor );
+
+		CKEditorInspector.attach( { editor: 'editor' } );
+
+		// Expose for playing in the console.
+		window.editor = editor;
+	} )
+	.catch( error => {
+		console.error( error.stack );
+	} );
```

Comment:
It's hard to tell what's going on here (and in other, similar snippets) - the change looks the same on the preview. Are those tabs changed to spaces or the other way around?

Reply:
Spaces to tabs. We use tabs in docs, globally.

Reply:
Ok, I think I see - it's deeper nesting.